flutter/lib/src/sentry_flutter.dart

```diff
@@ -243,6 +245,27 @@ mixin SentryFlutter {
     options.sdk = sdk;
   }
 
+  /// Screen redaction was previously introduced with the SessionReplay feature.
+  /// Screen redaction is enabled by default for SessionReplay.
+  /// As we also to use this feature for Screenshot, which previously was not
+  /// capable of redacting the screenshot, we need to disable redaction for Screenshot by default
+  /// so we don`t break the existing behavior.
+  /// As we have only one central place to configure the redaction,
+  /// we need to set the redaction options to full fill the above default settings.
+  /// The plan is to unify this behaviour with the next major release.
+  static void _setRedactionOptions(SentryFlutterOptions options) {
```

Comment:
```suggestion
  static void _setMaskingOptions(SentryFlutterOptions options) {
```

minor: since we changed the wording from redact -> mask, it might make sense to change it